docs/pages/attributesJSON.rst

```diff
@@ -177,6 +174,12 @@ Below are object-specific physical quantities.  These values will override simil
 "requires_lighting"
 	- boolean
 	- Whether the object should be rendered with lighting or flat shading.
+"is_visible"
+	- boolean
+	- Whether the object's render asset should be created upon instancing.
+"is_collidable"
+	- boolean
+	- Whether the object should be added to the collision and physics simulation world upon instancing. If false, object will be initialized with MotionType::VISUALIZATION_ONLY.
 "mass"
```

Comment:
It would be nice if there was some clean indication of what I need to specify or not for a VISUALIZATION_ONLY object. Do I need to choose a mass? A COM? margin? etc. It would be nice if these were all lumped into a "dynamics" section that could simply be omitted entirely for VISUALIZATION_ONLY objects.

Reply:
Good point. Yeah, you can specify any or none of these. Since a VISUALIZATION_ONLY is a MotionType, these objects can be enabled for physics simulation during runtime, so these properties could still be entirely relevant depending on the user's intentions.